open-api/rest-catalog-open-api.yaml

```diff
@@ -3014,28 +3060,26 @@ components:
           additionalProperties:
             type: string
 
-    PlanTableResult:
+    ScanTasks:
```

Comment:
Rather than repeating this, I've separated the scan and plan tasks into a separate type. I also added a `delete-files` list because that was missing.

Reply:
+1 to this idea